apps/bluetooth/js/startup_pair.js

```diff
@@ -4,15 +4,20 @@
 
 require(['config/require'], function() {
   // Bluetooth API version detect
+  console.log('[startup_pair]: AAA'); 
```

Comment:
This should be a debug message. Let's not output to the console directly. Please use a debug flag as usual.